packages/grid/src/vaadin-grid-column-resizing-mixin.js

```diff
@@ -57,9 +57,11 @@ export const ColumnResizingMixin = (superClass) =>
         const targetCell = columnRowCells.filter((cell) => cell._column === column)[0];
         // Resize the target column
         if (targetCell.offsetWidth) {
-          const style = window.getComputedStyle(targetCell);
+          // Resize handle uses pseudo-element as its clickable area
+          const handleStyle = getComputedStyle(handle, '::before');
+          const style = getComputedStyle(targetCell._content);
           const minWidth =
-            10 +
```

Comment:
Not sure if a problem, but this effectively applies a minimum allowed width of 67px (on Lumo) for resizable columns. If you add a column that has an initial width of, let's say 50px, once resized, it can no longer be shrunk back to its original width:

https://user-images.githubusercontent.com/1222264/159483767-7eee3d40-78ab-4927-9bb6-4dd1789bc3c7.mp4

Reply:
Good point, thanks. I think we would have to agree on some reasonable default for column min width then.
 
BTW if we use hardcoded value of `18` instead of `10`, with padding it would result in 50px (18 + 2 * 16).

Reply:
Can we check the explicitly defined width, and reduce the minimum width if needed based on that? Starts to get complicated, though.

Reply:
When resizing, we override the explicitly defined width by setting this property to a new value:
https://github.com/vaadin/web-components/blob/531a82cb01593dbb57d917684e2d71902e176147/packages/grid/src/vaadin-grid-column-resizing-mixin.js#L83
https://github.com/vaadin/web-components/blob/531a82cb01593dbb57d917684e2d71902e176147/packages/grid/src/vaadin-grid-column-resizing-mixin.js#L93

So, as soon as the column is resized once, the defined width for this column and all preceding columns gets lost.

Reply:
IMO the ideal solution would be to add new property e.g. `minWidth` with a reasonable value (e.g. `50px`).
We could then use that for calculating min width when resizing to make it straightforward. WDYT?